packages/components/src/navigator/navigator-back-button/hook.ts

```diff
@@ -12,10 +12,11 @@ import useNavigator from '../use-navigator';
 import type { NavigatorBackButtonProps } from '../types';
 
 export function useNavigatorBackButton(
-	props: WordPressComponentProps< NavigatorBackButtonProps, 'button' >
+	props: WordPressComponentProps< NavigatorBackButtonProps, 'button' > | any
```

Comment:
just a temporary hack of course until we decide if `href` should be added to this component or not.